src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -3260,6 +3268,16 @@ static void _svgLoaderParserXmlOpen(SvgLoaderData* loader, const char* content,
         if (loader->stack.count > 0) parent = loader->stack.last();
         else parent = loader->doc;
         node = method(loader, parent, attrs, attrsLength, simpleXmlParseAttributes);
+        if (node && !empty) {
+            auto defs = _getDefsNode(node);
+            if (!defs) {
+                defs = _createNode(nullptr, SvgNodeType::Defs);
+                loader->def = defs;
+                loader->doc->node.doc.defs = defs;
+            }
+            loader->stack.push(defs);
```

Comment:
I'd replace these lines:
```
            auto defs = _getDefsNode(node);
            if (!defs) {
                defs = _createNode(nullptr, SvgNodeType::Defs);
                loader->def = defs;
                loader->doc->node.doc.defs = defs;
            }
```
by 
```
            auto defs = _createDefsNode(loader, nullptr, nullptr, 0, nullptr);
```
but I leave the decisions to you

Reply:
Ooh.. i didn't add comment in `defs`. I will write this in the next.